sessions/backend-api-routes/products/services/productServices.js

```diff
@@ -0,0 +1,67 @@
+const products = [
+  {
+    id: "1",
+    name: "Shrimp",
+    description: "Prefers to live in pairs",
+    price: 19,
+    currency: "€",
+    category: "Invertebrates",
```

Comment:
IIRC we're usually giving our students data they can work with easily. Should they ever have data that might be a little more challenging to work with? Like having to convert floats to ints, etc. I figure this might be more leaning toward working with types, just wondering.